.github/workflows/js.yml

```diff
@@ -3,7 +3,6 @@ name: JS CI
 
 on:
   push:
-    branches: ["main"]
```

Comment:
Just making the js tests run on every commit so you don't have to open a PR to see them.

Reply:
Love it!